sdk/storage/azure-storage-file-datalake/src/main/java/com/azure/storage/file/datalake/models/PathDeletedItem.java

```diff
@@ -0,0 +1,82 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.storage.file.datalake.models;
+
+import java.time.OffsetDateTime;
+
+/**
+ * A path that has been soft deleted.
+ */
+public class PathDeletedItem {
+    private String name;
+    private boolean isPrefix;
+    private String deletionId;
+    private OffsetDateTime deletedOn;
+    private Integer remainingRetentionDays;
+
+    /**
+     * Constructs a {@link PathDeletedItem}.
+     *
+     * @param name The name of the path
+     * @param isPrefix Whether the item is a prefix
+     * @param deletionId The deletion id associated with the deleted path to uniquely identify it from other items
+     * deleted at this path
+     * @param deletedOn When the path was deleted
+     * @param remainingRetentionDays The number of days left before the soft deleted path will be permanently deleted
+     */
+    public PathDeletedItem(String name, boolean isPrefix, String deletionId, OffsetDateTime deletedOn,
```

Comment:
Could we add like isDirectory to this? That might be important info a customer would want

Reply:
Isn't that the same as isPrefix? Maybe I should just change the name of that?